Fix 'push to remote name not origin test'

seanodell opened the pull request:
This test was failing only because the branch and remote names were single quoted. I don't know if they fail for everyone; may not be a useful PR if this just breaks for other people.

webpro commented:
The tests are not failing in this test matrix: https://travis-ci.org/release-it/release-it, so I'm a bit surprised it fails for you. What OS are you using?

seanodell commented:
MacOS High Sierra with git 2.14.1.

```
Seans-MacBook-Pro-3:release-it Sean$ git remote add upstream git@github.com:seanodell/release-it.git
Seans-MacBook-Pro-3:release-it Sean$ git checkout -b foo
Switched to a new branch 'foo'
Seans-MacBook-Pro-3:release-it Sean$ git push --follow-tags  -u upstream foo
Branch foo set up to track remote branch foo from upstream.
Everything up-to-date
Seans-MacBook-Pro-3:release-it Sean$ git --version
git version 2.14.1
```

I upgraded git and now get this:

```
Seans-MacBook-Pro-3:release-it Sean$ git remote add upstream git@github.com:seanodell/release-it.git
Seans-MacBook-Pro-3:release-it Sean$ git checkout -b foo
Switched to a new branch 'foo'
Seans-MacBook-Pro-3:release-it Sean$ git push --follow-tags  -u upstream foo
Branch 'foo' set up to track remote branch 'foo' from 'upstream'.
Everything up-to-date
Seans-MacBook-Pro-3:release-it Sean$ git --version
git version 2.21.0
```

I don't think there is a standard to git outputs. Might be better to accept either with or without the single quotes.

webpro commented:
Thanks for the output, I've merged your changes and released this in [v10.4.4](https://github.com/release-it/release-it/releases/tag/10.4.4) (btw, also cherry-picked the dummy GitHub/GitLab tokens during test from v11).
